biomage/experiment/download.py

```diff
@@ -0,0 +1,306 @@
+import json
+import os
+from enum import Enum
+from fileinput import filename
+from pathlib import Path
+
+import boto3
+import click
+from botocore.exceptions import ClientError
+
+from ..rds.run import run_rds_command
+from ..utils.constants import (
+    CELLSETS_BUCKET,
+    PROCESSED_FILES_BUCKET,
+    RAW_FILES_BUCKET,
+    SAMPLES_BUCKET,
+)
+from .utils import set_modified_date
+
+output_path = "."
+
+
+s3 = boto3.resource("s3")
```

Comment:
you have `s3` as a global value, but you only use it inside `_download_file` function. I suggest to better move the declaration of s3 to be inside that function